src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -455,10 +455,13 @@ export default class FilterOptionsComponent extends Component {
               if (matchedSubstring) {
                 filterOption.classList.add('displaySearch');
                 filterOption.classList.remove('hiddenSearch');
-                labelEl.textContent = new HighlightedValue({
+                const highlightedElements = new HighlightedValue({
                   value: labelText,
                   matchedSubstrings: [matchedSubstring]
-                }).get();
+                }).getElements();
+
+                labelEl.textContent = '';
```

Comment:
do we need to set the textContent to an empty string? or is it sufficient to just append the elements in the next line?

Reply:
it's used to clear the previous value